wp-parsely.php

```diff
@@ -163,7 +211,12 @@ function maybe_load_plugin() {
 	// No integration: The site has not enabled parsely.
 	if ( ! $should_load || $should_prevent_loading ) {
 		Parsely_Loader_Info::set_active( false );
-		Parsely_Loader_Info::set_integration_type( Parsely_Loader_Info::INTEGRATION_TYPE_NONE );
+
+		if ( false === $filtered_load_status ) {
+			Parsely_Loader_Info::set_integration_type( Parsely_Integration_Type::DISABLED_MUPLUGINS_FILTER );
+		} elseif ( '0' === $option_load_status ) {
+			Parsely_Loader_Info::set_integration_type( Parsely_Integration_Type::DISABLED_MUPLUGINS_SILENT_OPTION );
+		}
```

Comment:
Are these the only two possibilities? Should there be a fallback to `INTEGRATION_TYPE_NONE` in case neither conditional happens?

Reply:
In [get_integration_type](https://github.com/Automattic/vip-go-mu-plugins/blob/sends-updated-parsely-config-states-to-SDS/wp-parsely.php#L50) function there is already a fallback to select `NONE` if there is no integration type.